include/bluetooth/enocean.rst

```diff
@@ -0,0 +1,112 @@
+.. _bt_enocean_readme:
+
+Bluetooth EnOcean
+#################
+
+The Bluetooth EnOcean library allows passive observation of BLE-based wall switches and sensors from `EnOcean`_, such as the Push Button Transmitter Module 215B wall switch module (PTM 215B) and the Easyfit Motion Detector with Illumination Sensor module (EMDCB sensor).
+This means that you can only observe the output of the devices, without sending anything.
+Although the EnOcean devices offer the two-way interaction, this library does not support it.
+
+The library supports different types of commissioning for both wall switches and sensors:
+
+    * radio-based commissioning,
+    * manually inputting data from other commissioning methods, such as NFC or QR codes.
+
+The library supports replay protection based on sequence numbers.
+Only new, authenticated packets from commissioned devices will go through to the event callbacks.
+
+If the :option:`CONFIG_BT_SETTINGS` subsystem is enabled, the device information of all commissioned devices are stored persistently using the :ref:`zephyr:settings` subsystem, including the sequence number information.
+
+For the demonstration of the library features, see the :ref:`enocean_sample` sample.
+
+.. _bt_enocean_devices:
+
+Supported devices
+=================
+
+This library supports both PTM 215B-based switches and the BLE-enabled sensor module:
+
+* `Easyfit Single/Double Rocker Wall Switch for BLE - EWSSB/EWSDB`_
+* `Easyfit Single/Double Rocker Pad for BLE - ESRPB/EDRPB`_
+* `Easyfit Motion Detector with Illumination Sensor - EMDCB`_
+
+Single rocker switches work differently with the library than double rocker devices.
```

Comment:
This is inaccurate. Single rocker switches work the same way, but they only engage switch 3 and 4. I don't think it's necessary to mention this here, really.